src/ISeabotConfig.ts

```diff
@@ -0,0 +1,14 @@
+export default interface ISeabotConfig {
+    userVoiceChannels: {
+        groupId: string;
+        triggerChannelId: string;
+    };
+    autoDeleteMessagesInChannels: Array<{ id: string; frequency: number }> | undefined;
+    telemetryChannels: Array<string> | undefined;
```

Comment:
Are these `telemetryChannels` or `telemetryChannelCategories`?

Reply:
That actually brings up a good question...currently it's a list of categories, but do we want to support individual channels as well? If so, we need a variable name that encompasses both without being too verbose (or split into telemetryChannels AND categories)

Reply:
I went ahead and implemented this to use both channels and categories, since it was fairly easy to cover both.